docs/source/advanced/multi-task-learning.rst

```diff
@@ -0,0 +1,266 @@
+                        "task_type": "classification",
+                        "split_pct":	[0.8, 0.1, 0.1],
+                        "mask_field_names": ["train_mask_class",
+                                             "val_mask_class",
+                                             "test_mask_class"]
+                    }
+                ]
+            },
+
+                ......
+
+        ],
+        "edges": [
+
+            ......
+
+            {
+                "relation": [
+                    "paper",
+                    "citing",
+                    "paper"
+                ],
+                "format": {
+                    "name": "parquet"
+                },
+                "files": [
+                    "/tmp/acm_raw/edges/paper_citing_paper.parquet"
+                ],
+                "source_id_col": "source_id",
+                "dest_id_col": "dest_id",
+                "labels": [
+                    {
+                        "task_type": "link_prediction",
+                        "split_pct":	[0.8, 0.1, 0.1],
+                        "mask_field_names": ["train_mask_lp",
+                                             "val_mask_lp",
+                                             "test_mask_lp"]
+                    }
+                ]
+            },
+
+        ......
+
+        ]
+    }
+
+In the above configuration, we define one task for the **paper** node and one task for the
+**paper,citing,paper** edge. The node classification task will take the label name of `label_class` and the train/validation/test mask fields as `train_mask_class`,
+`val_mask_class` and `test_mask_class`, respectively. The link prediction task will take the train/validation/test mask fields as `train_mask_lp`, `val_mask_lp` and `test_mask_lp`, respectively.
+
+
+Construct Graph
+~~~~~~~~~~~~~~~~
+You can follow the instructions in :ref:`Run graph construction<run-graph-construction>` to use the
+GraphStorm construction tool for creating partitioned graph data for training. Please ensure you
+customize the command line arguments such as `--conf-file`, `--output-dir`, `--graph-name` to your
+specific values.
+
+
+Run Multi-task Learning Training
+--------------------------------
+Running a multi-task learning training task is similar to running other GraphStorm built-in tasks as
+detailed in :ref:`Launch Training<launch-training>`. The main defines is to define multiple training
+targets in the YAML configuration file.
+
+
+Define Multi-task for training
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can specify multiple training tasks for a training job by providing the `multi_task_learning`
+configurations in the yaml file The following config defines two training tasks, one for node
+classification and one for edge classification.
+
+.. code-blocks:: yaml
+
+    ---
+    version: 1.0
+    gsf:
+        basic:
+            ...
+        ...
+        multi_task_learning:
+            - node_classification:
+                target_ntype: "movie"
```

Comment:
Because the above JSON file use the ACM data as examples, can we also use ACM data as examples here to help users to better understand.